### DISCUSSION TITLE
Doris 3.0存算分离文档不合

### DISCUSSION THREAD

**Discussion by owenchenxy** [apache/doris#43101] [A - General / Q&A]
https://doris.apache.org/zh-CN/docs/3.0/install/cluster-deployment/k8s-deploy/compute-storage-decoupled/install-quickstart

这个链接的文档， 从一开始apply crd资源的预期结果开始就与实际执行不符， 后面的curl的link也都不存在，怎么用3.0存算分离， 有大佬成功吗

**Comment by catpineapple** [comment]
文档已经更新，存算分离 operator 的 CRD 和之前的版本变更较大，以当前版本为准，doris 版本使用大于 3.0.2 的 @owenchenxy
